### PR TITLE
Add FaceAPI PersonDirectory spec updates to preview v1.0

### PR DESCRIPTION
**Pull request by msyache**
Add a preview version of FaceAPI specs intended for public preview.
Contains existing spec in v1.0 plus PersonDirectory feature changes.

<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Changelog
Please ensure to add changelog with this PR by answering the following questions.
  1. What's the purpose of the update?    
      - [ ] new service onboarding 
      - [x] new API version 
      - [ ] update existing version for new feature 
      - [ ] update existing version to fix swagger quality issue in s360
      - [ ] Other, please clarify 
  2. When you are targeting to deploy new service/feature to public regions? Please provide date, or month to public if date is not available yet.
  -  4/5
  3. When you expect to publish swagger? Please provide date, or month to public if date is not available yet.
   - 4/12
  4. If it's an update to existing version,  please select SDKs of specific language and CLIs that require refresh after swagger is published.
      - [ ] SDK of .NET (need service team to ensure code readiness)
      - [ ] SDK of Python
      - [ ] SDK of Java
      - [ ] SDK of Js
      - [ ] SDK of Go
      - [ ] PowerShell
      - [ ] CLI
      - [ ] Terraform
      - [ ] No, no need to refresh for updates in this PR

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of "no breaking changes"
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  -  [ ] Ensure to copy the existing version into new directory structure for first commit (including refactoring) and then push new changes including version updates in separate commits. This is required to review the changes efficiently.
  - Adding a new service

- [ ] Please ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from Breaking Change Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable or public preview version with Breaking Change Validation errors
- [ ] Updating API(s) in public preview over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

File: specification/cognitiveservices/data-plane/Face/readme.md [hunk 1]
```diff
@@ -21,6 +21,13 @@ These settings apply only when `--tag=release_1_0` is specified on the command l
 input-file: stable/v1.0/Face.json
 ```
 
+### Release 1.0-preview
```
Comment (jhendrixMSFT) [inline specification/cognitiveservices/data-plane/Face/readme.md:24]: I'm a little confused on this versioning.  Seems like a v1.0-preview would come before a stable v1.0 which already exists.  Can you please clarify?

Comment (msyache) [reply specification/cognitiveservices/data-plane/Face/readme.md:24]: The case for this feature is a little special as it is approved by API review board to go into Face API stable v1.0 after going GA. As there is no next target stable version, we figured v1.0-preview would best convey the intention.

Comment (jhendrixMSFT) [reply specification/cognitiveservices/data-plane/Face/readme.md:24]: Thanks for clarifying.  Can I diff this against the v1.0 stable to see the delta?

Comment (jhendrixMSFT) [reply specification/cognitiveservices/data-plane/Face/readme.md:24]: OK comparing against v1.0 does the trick.

Does this mean that this version will replace the current v1.0 stable?

File: specification/cognitiveservices/data-plane/Face/preview/v1.0-preview/Face.json [hunk 2]
```diff
@@ -0,0 +1,5548 @@
...
+        }
+      }
+    },
+    "/persons": {
+      "get": {
+        "tags": [
+          "PersonDirectory"
+        ],
+        "summary": "Retrieve list of person information in person directory.",
+        "operationId": "PersonDirectory_GetPersons",
```
Comment (jhendrixMSFT) [inline specification/cognitiveservices/data-plane/Face/preview/v1.0-preview/Face.json:2521]: I'm a bit confused by this operation.  The name and summary implies it returns a collection however the response schema is singular.  Can you please clarify?

File: specification/cognitiveservices/data-plane/Face/readme.md (hunk 1 above)
Comment (msyache) [reply specification/cognitiveservices/data-plane/Face/readme.md:24]: When the diff (Person Directory feature) eventually goes into GA, yes.

File: specification/cognitiveservices/data-plane/Face/preview/v1.0-preview/Face.json (hunk 2 above)
Comment (msyache) [reply specification/cognitiveservices/data-plane/Face/preview/v1.0-preview/Face.json:2521]: The response returns an array of the schema item.

File: specification/cognitiveservices/data-plane/Face/preview/v1.0-preview/Face.json [hunk 3]
```diff
@@ -0,0 +1,5548 @@
...
+        "parameters": [
+          {
+            "$ref": "#/parameters/operationId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "A successful call returns the operation's status.",
+            "schema": {
+              "$ref": "#/definitions/AsyncStatus"
```
Comment (jhendrixMSFT) [inline specification/cognitiveservices/data-plane/Face/preview/v1.0-preview/Face.json:2495]: I see this changed from v1.0.  Does this need the [x-ms-long-running-operation](https://github.com/Azure/autorest/tree/master/docs/extensions#x-ms-long-running-operation) annotation?

Comment (msyache) [reply specification/cognitiveservices/data-plane/Face/preview/v1.0-preview/Face.json:2495]: The updated schema is still compatible with the previous version, just with some new fields.
Regarding the x-ms-long-running-operation annotation, it probably would have made sense to be added to the operations that actually produce 202s, but given that FaceAPI has been following the [API guidelines to use Operation-Location](https://github.com/Microsoft/api-guidelines/blob/master/Guidelines.md#1321-put) in the header, it seems incompatible with the final-state-via definitions. For the sake of consistency with existing long-running operations in the service, we would prefer to keep it the current way for now.

**Review by jhendrixMSFT — APPROVED** [review]

**Pull request merged**